cpp/src/parquet/parquet.thrift

```diff
@@ -17,11 +17,11 @@
  * under the License.
```

Comment:
this is just copying from the new release?

Reply:
Yes!